<ui version="4.0" >
 <class>NetworkDialog</class>
 <widget class="QDialog" name="NetworkDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>412</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Network</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>ESSID:</string>
     </property>
     <property name="buddy" >
      <cstring>essidEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="essidEdit" >
     <property name="maxLength" >
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>BSSID:</string>
     </property>
     <property name="buddy" >
      <cstring>bssidEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="bssidEdit" >
     <property name="maxLength" >
      <number>17</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Channel:</string>
     </property>
     <property name="buddy" >
      <cstring>channelSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QSpinBox" name="channelSpin" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>12</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Encryption:</string>
     </property>
     <property name="buddy" >
      <cstring>encryptionCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QComboBox" name="encryptionCombo" >
     <item>
      <property name="text" >
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>WEP</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>WPA</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>WPA2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Latitude:</string>
     </property>
     <property name="buddy" >
      <cstring>latitudeSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QDoubleSpinBox" name="latitudeSpin" >
     <property name="decimals" >
      <number>6</number>
     </property>
     <property name="minimum" >
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum" >
      <double>90.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Longitude:</string>
     </property>
     <property name="buddy" >
      <cstring>longitudeSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QDoubleSpinBox" name="longitudeSpin" >
     <property name="decimals" >
      <number>6</number>
     </property>
     <property name="maximum" >
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>Comment:</string>
     </property>
     <property name="buddy" >
      <cstring>commentEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2" >
    <widget class="QPlainTextEdit" name="commentEdit" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_9" >
     <property name="text" >
      <string>Signal:</string>
     </property>
     <property name="buddy" >
      <cstring>signalSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QSpinBox" name="signalSpin" >
     <property name="maximum" >
      <number>100</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>essidEdit</tabstop>
  <tabstop>bssidEdit</tabstop>
  <tabstop>channelSpin</tabstop>
  <tabstop>signalSpin</tabstop>
  <tabstop>encryptionCombo</tabstop>
  <tabstop>latitudeSpin</tabstop>
  <tabstop>longitudeSpin</tabstop>
  <tabstop>commentEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NetworkDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>219</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel" >
     <x>276</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NetworkDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>102</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel" >
     <x>115</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
